multiworld/envs/mujoco/cameras.py

```diff
@@ -164,6 +164,21 @@ def sawyer_door_env_camera_v2(camera):
     camera.azimuth = cam_pos[5]
     camera.trackbodyid = -1
 
+def sawyer_door_env_camera_v3(camera):
```

Comment:
For future reference, can you just make this like the other functions, which just sets the variables (no for loops)?